layouts/basic/modules/Settings/BusinessHours/EditViewBlocks.tpl

```diff
@@ -0,0 +1,92 @@
+{*<!--
```

Comment:
`{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}`